card/mod/tinymce_editor/lib/javascript/script_tinymce_config.js.coffee

```diff
@@ -19,17 +19,20 @@ $.extend decko,
     # verify_html: false -- note: this option needed for empty
     #                             paragraphs to add space.
     conf = {
+      theme: "modern",
       plugins: 'autoresize'
-      autoresize_max_height: 500
+      autoresize_max_height: 500,
     }
     user_conf = if decko.tinyMCEConfig? then decko.tinyMCEConfig else {}
     hard_conf = {
-      mode: 'exact'
-      elements: el_id
+      selector: "##{el_id}",
```

Comment:
inconsistent commas in this file (here and above).  I think coffeescript convention is no comma.